core/src/test/java/com/github/jsonldjava/utils/JsonUtilsTest.java

```diff
@@ -15,6 +17,19 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 public class JsonUtilsTest {
+    @Test
+    public void resolveTest() {
+        final String baseUri = "http://mysite.net";
+        final String pathToResolve = "picture.jpg";
+        String resolve = "";
+
+        try {
+            resolve = JsonLdUrl.resolve(baseUri, pathToResolve);
+        } catch (final Exception e) {
+            assertTrue(false);
```

Comment:
The usual pattern for this is to have a fail at the end, and put the assertEquals inside of the try-catch block. I know the next method does it this way, but it isn't ideal.

Reply:
@ansell please have a look if  https://github.com/jsonld-java/jsonld-java/pull/280/commits/9359f10b89b9a16b7670ccb0ee0beafb0295e3bd does what you've suggested.

Reply:
Looks okay now. I may get to improving the overall style someday, but this is good enough for today.